src/cs/Ssh.Tcp/SshServer.cs

```diff
@@ -222,7 +222,15 @@ protected virtual void Dispose(bool disposing)
 	{
 		if (disposing)
 		{
-			this.disposeCancellationSource.Cancel();
+			try
+			{
+				this.disposeCancellationSource.Cancel();
```

Comment:
If the code doesn't use wait handle from `disposeCancellationSource`, there is no need to dispose it and catch an exception here.